StudyRingMeteor/collections/rings/rings.js

```diff
@@ -85,6 +85,19 @@ RingSchema = new SimpleSchema({
         autoform: {
             type: "hidden"
         }
+    },
+    location: {
+         type: String,
+         label: "location",
+        autoValue: function() {
+            //This makes sure to only set a value when it is an insert function, not an update
+            if (this.isInsert && (!this.isSet || this.value.length === 0)) {
+                return new Array();
+            }
+        },
+        autoform: {
+            type: "hidden"
```

Comment:
Do not make it hidden or else, event location will not be possible to set during its creation